lib/order.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/menu.rb'
+
+class Order
+  attr_reader :basket
+
+  def initialize(menu = Menu.new)
+    @basket = Hash.new(0)
+    @menu = menu
+  end
+
+  def add_order(dish, quantity)
+    @basket[dish.to_sym] += quantity
+    output = "#{quantity}x #{dish}(s) added to your basket"
+    output
+  end
+
+  def basket_summary
+    raise "Your basket is empty!" if basket.empty?
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)